tests/callbacks/test_callback_requests.py

```diff
@@ -32,11 +32,7 @@
 from airflow.utils import timezone
 from airflow.utils.state import State
 
-TI = TaskInstance(
```

Comment:
Comment for self: This one hits DB while collecting tests, so you cannot run test collection with `--skip-db-tests`. What works is moving the initialization of ORM models to inside the test.